stroyprombeton/settings/base.py

```diff
@@ -145,7 +145,7 @@
 STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.ManifestStaticFilesStorage'
 
 STATICFILES_DIRS = [
-    os.path.join(BASE_DIR, 'front/build'),
+    os.path.join(BASE_DIR, 'build'),
```

Comment:
Why did you update `front/build` to `build`? As I remember we store built static files in `front/build`. Also we have an [assert](https://github.com/fidals/stroyprombeton/blob/master/docker/images/python/Dockerfile.prod#L10) to check existence of these files.

Seems this [issue](https://github.com/fidals/stroyprombeton/issues/257) occurs because built static files do not exist at `collectstatic` step cause of broken `nodejs` image

Reply:
@artemiy312 , we moved `front/build` dir to `build` to prevent volumes conflicts

https://github.com/fidals/stroyprombeton/blob/master/docker/docker-compose.yml#L42

Reply:
@duker33 It is relevant only for `nodejs` service, but not for `app`.
`app` still stores build static in `front/build`

Reply:
@artemiy312 , wow, it's true for `dcp.yml`, but not true for `dc.yml`.
I'll create pdd task

Reply:
Done.